examples/eqpt_config.json

```diff
@@ -0,0 +1,43 @@
+{     "Edfa":[{
+            "type_variety": "std_medium_gain",
+            "gain_flatmax": 25,
+            "gain_min": 15,
+            "p_max": 21,
+            "advanced_config_from_json": "std_medium_gain_advanced_config.json"
+            },
+            {
+            "type_variety": "std_low_gain",
+            "gain_flatmax": 19,
+            "gain_min": 12,
+            "p_max": 22,
+            "nf_min": 6.5,
+            "nf_max": 10
+            }
+      ],
+      "Fiber":[{
```

Comment:
I presume these are arrays of maps, so that you can specify subtypes of "Fiber."

Could this be a simple map of maps?

```
"Fiber": { "SSMF": { ... }, ... }
```

Reply:
Just keeping the same convention than for the edfa_eqpt config and topology json